productive-seas-fish-and-shellfish-stocks-fished-sustainably/wrangling.py

```diff
@@ -0,0 +1,19 @@
+from email.policy import default
+import click
+import pandas as pd
+from pathlib import Path
+
+@click.command()
+@click.argument("input", type=click.Path(exists=True, path_type=Path))
+@click.option("--output", default=Path("./output.csv"), type=click.Path(path_type=Path))
+def wrangle(input: Path(), output: Path()) -> None:
+    
```

Comment:
It is also good to arrange the dataframe columns in the order in which they should appear